modules/exploits/linux/http/vmware_workspace_one_access_cve_2022_22954.rb

```diff
@@ -47,7 +47,7 @@ def initialize(info = {})
               'Arch' => ARCH_CMD,
               'Type' => :cmd,
               'DefaultOptions' => {
-                'PAYLOAD' => 'cmd/unix/reverse_bash'
+                'PAYLOAD' => 'cmd/unix/python/meterpreter/reverse_tcp'
```

Comment:
If this is the payload you'd like used as the default, I'm pretty sure it's the one that will automatically be selected by the framework.

Reply:
Doesn't change with target selection, though, unlike https://github.com/rapid7/metasploit-framework/pull/15874#discussion_r755178475.

Reply:
Until it's fixed, we'll want to leave this so the right default is set.